src/test/isolation2/sql/pg_rewind_fail_missing_xlog.sql

```diff
@@ -5,6 +5,27 @@
 CREATE EXTENSION IF NOT EXISTS gp_inject_fault;
 include: helpers/server_helpers.sql;
 
+CREATE OR REPLACE LANGUAGE plpythonu;
+CREATE OR REPLACE FUNCTION connectSeg(n int, port int, hostname text) RETURNS bool AS $$
+import os
+import subprocess
+import time
+for i in range(n):
+    try:
+        cmd = 'PGOPTIONS="-c gp_session_role=utility" psql -h %s -p %d -d postgres -Xc "select 1"' % (hostname, port)
+        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+        stdout, stderr = proc.communicate()
+        if proc.returncode == 0:
+            return True
+        else:
+            raise Exception("%s:%d stdout=%s, stderr=%s" % (hostname, port, stdout, stderr))
+    except Exception as e:
+        plpy.warning(e)
```

Comment:
Why warning is added? it wasn't in the original patch. Could this affect the passing of the test?

Reply:
yes, you are right, updated